src/lambdas/auditMessageFirehoseReingest/sendAuditEventsToFirehose.ts

```diff
@@ -0,0 +1,75 @@
+import { PutRecordBatchOutput } from '@aws-sdk/client-firehose'
+import { firehosePutRecordBatch } from '../../services/firehose/firehosePutRecordBatch'
+import { logger } from '../../services/logger'
+import { AuditEvent } from '../../types/auditEvent'
+import { S3ObjectDetails } from '../../types/s3ObjectDetails'
+import { auditEventsToFirehoseRecords } from '../../utils/helpers/firehose/auditEventsToFirehoseRecords'
+import { getEnv } from '../../utils/helpers/getEnv'
+
+export const sendAuditEventsToFirehose = async (
+  s3ObjectDetails: S3ObjectDetails[]
+): Promise<S3ObjectDetails[]> =>
+  Promise.all(
+    s3ObjectDetails.map(async (details) => {
```

Comment:
Wherever possible, I'd prefer to be able to see exactly what's going on in a map with a nicely named function, think that's possible here

Reply:
Done